src/OpenTK/Platform/X11/X11GLNative.cs

```diff
@@ -839,7 +910,114 @@ public override void ProcessEvents()
                                 break;
                             }
                         }
-                        
+                        else if (e.ClientMessageEvent.message_type == _atom_xdnd_enter)
+                        {
+                            // Xdnd started
+                            bool useList = ((e.ClientMessageEvent.ptr2.ToInt64() & 1) == 1);
+                            sourceHandler = e.ClientMessageEvent.ptr1;
+                            sourceXdndVersion = e.ClientMessageEvent.ptr2.ToInt64() >> 24;
```

Comment:
Just throw in a comment above the line explaining what the shift does and where you got the value from and I'm happy :)